sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -5,13 +5,51 @@
 ```ts
 
 import { CommonClientOptions } from '@azure/core-client';
+import * as coreClient from '@azure/core-client';
 import { OperationOptions } from '@azure/core-client';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { TokenCredential } from '@azure/core-auth';
 
 // @public
 export type AggregationType = "None" | "Average" | "Count" | "Minimum" | "Maximum" | "Total";
 
+// @public
+export interface BatchMetadataValue {
+    name?: LocalizableString;
+    value?: string;
+}
+
+// @public
+export interface BatchMetric {
+    displayDescription: string;
+    errorCode?: string;
+    errorMessage?: string;
+    id: string;
+    name: LocalizableString;
+    timeseries: BatchTimeSeriesElement[];
+    type: string;
+    unit: BatchMetricUnit;
+}
+
+// @public
+export type BatchMetricUnit = "Count" | "Bytes" | "Seconds" | "CountPerSecond" | "BytesPerSecond" | "Percent" | "MilliSeconds" | "ByteSeconds" | "Unspecified" | "Cores" | "MilliCores" | "NanoCores" | "BitsPerSecond";
+
+// @public
+export interface BatchMetricValue {
+    average?: number;
+    count?: number;
+    maximum?: number;
+    minimum?: number;
+    timeStamp: Date;
+    total?: number;
+}
+
+// @public
+export interface BatchTimeSeriesElement {
```

Comment:
Can we reuse the existing `TimeSeriesElement` type returned in the single query response? Also, there are other models from single query response that already exist that can be reused.

Reply:
All possible values have been reused except `Metric`. I have renamed the `Metric` from batch operation to `BatchQueryMetric`. Why? Because, the `name` property is of different type in the old and new ones. In the old one, it is of data type `string`. In the new one, it is `LocalizedString`. All the other models have been reused.